tooling/api/src/fs.ts

```diff
@@ -367,7 +367,7 @@ export type {
 }
 
 export {
-  BaseDirectory as Dir,
```

Comment:
we need to keep this because it's a breaking change :/

Reply:
aaah you're right, will revert it back